### PR TITLE
Next round of doc changes.

### PR DESCRIPTION
**Pull request by slackmoehrle**

### CODE REVIEW

**Comment by slackmoehrle** [comment]
@minggo Here are topics that I am adding: common game engine terms, Sqlite, multi-resolution, using the prebuilt libraries. What else should we add? Are there specific 3.16 pieces I should write content for?

**Comment by minggo** [comment]
I am not sure if we should add a section saying about the creator plugin for cocos2d-x: https://github.com/cocos2d/creator_to_cocos2dx.

**Comment by slackmoehrle** [comment]
Let’s add it!  I will review how to use it and write up a draft for review.

________________________________

**Comment by minggo** [comment]
Sounds good, thanks.

**Comment by slackmoehrle** [comment]
@minggo where do you think the creator_to_cocos2d-x plugin should go in our docs?

**Comment by minggo** [comment]
How about add a section for `editor support`?

**Comment by slackmoehrle** [comment]
ok, you got it. Can you tell me if these new docs should include Creator docs like it does now? 

http://cocos2d-x.org/docs/editors_and_tools/creator/

I haven't updated these in forever, so perhaps I should pull new ones too. I should probably integrate with what Nantas does so I can automate this for the future. The Creator docs in the link I manually built :-(

**Comment by minggo** [comment]
No, i don't think we need to include these docs in cocos2d-x's docs, may be just add a link somewhere is enough.

**Comment by slackmoehrle** [comment]
ok. Thanks. In the Header should be enough. I will modify the header to make more sense for us. 

screen shot 2017-10-11 at 9 34 11 pm

File: en/editors_and_tools/creator_to_cocos2dx.md [hunk 1]
```diff
@@ -3,3 +3,18 @@
 </div>
 
 ## Using Cocos Creator With C++ and Lua Projects
+__Cocos Creator__ supports JavaScript, built in. Edit your `Scenes` and source code all from within. However, If you are a C++ or Lua developer, __Creator__ allows exporting of `Scenes` to sour code for further development. Why isn't C++ built in, you ask? There is no need to re-invent the wheel. There are many really good development environments out there. Writing a text editor is no trivial task. For this reason, it is best to allow developers to edit source code in the editor of their choice.
+
+### What Is Supported?
```
Comment (slackmoehrle) [inline en/editors_and_tools/creator_to_cocos2dx.md]: Take a look at these headings. Am I covering everything?

Comment (minggo) [reply en/editors_and_tools/creator_to_cocos2dx.md]: I think these things can be added too:

- how to run the demo project that i have done
- how to integrate exported source code and resources in new projects
- how to use AnimationManager
- how to use ColliderManager
- what cocos2d-x versions and Creator versions supported

So may be there should be more docs for it.

Comment (minggo) [reply en/editors_and_tools/creator_to_cocos2dx.md]: I mean, this doc is good for basic introduction, but may need more docs about it.

Comment (slackmoehrle) [reply en/editors_and_tools/creator_to_cocos2dx.md]: @minggo Yup, I am adding all of that.

Comment (minggo) [reply en/editors_and_tools/creator_to_cocos2dx.md]: Awesome. Please let me know if you need any help.
Thanks.

**Pull request merged**